packages/@tinacms/graphql/src/index.ts

```diff
@@ -32,26 +31,23 @@ export type DummyType = unknown
 export const buildSchema = async (
```

Comment:
Does this need to live in the GraphQL package? I'm assuming it's only used by the CLI